Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -198,3 +198,25 @@ instance {L : Type*} [Field k] [Field L] [Algebra k L] [Algebra.IsAlgebraic k L]
   isAlgClosed := inferInstance
 
 end AlgebraicClosure
+
+namespace IntermediateField
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L] (E : IntermediateField K L)
+
+theorem AdjoinSimple.normal_algebraicClosure {x : L} (hx : IsIntegral K x) :
+    Normal K (AlgebraicClosure K⟮x⟯) :=
+  have : IsAlgClosure K (AlgebraicClosure ↥K⟮x⟯) := by
```

Comment:
Maybe this should be a separate lemma?

Reply:
I forgot to reply to this comment. I had added `AdjoinSimple.is_algebraicClosure` for this, which I have now generalized and made an instance following @erdOne's suggestion.